From 1147cf3fe1c0cbf7494d1908196c8cfe599c8f2a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:55 -0500
Subject: [redhat] kabi: Adding symbol paravirt_ticketlocks_enabled (arch/x86/kernel/paravirt-spinlocks.c) X86 ARCHITECTURE (32-BIT AND 64-BIT)

Message-id: <92540924fc72f7fd320ed2bce65c6d0d036cf593.1416318767.git.poros@redhat.com>
Patchwork-id: 100177
O-Subject: [PATCH RHEL7.1 50/63] kabi: Adding symbol paravirt_ticketlocks_enabled (arch/x86/kernel/paravirt-spinlocks.c) X86 ARCHITECTURE (32-BIT AND 64-BIT)
Bugzilla: 1070274
RH-Acked-by: Andrew Jones <drjones@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    These three symbols seem to be dragged indirectly by paravirt.
    I cannot immediately reproduce why; in any case there probably is nothing
    we can do about it anyway, though.
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 101/417] kabi: Adding symbol paravirt_ticketlocks_enabled (arch/x86/kernel/paravirt-spinlocks.c)
 Jiri Olsa <jolsa@redhat.com> (Wed, 12 Feb 2014 15:50:21 +0100)
 Description:
 ============
 It is called in spinlock operations.

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/paravirt_ticketlocks_enabled

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/paravirt_ticketlocks_enabled b/redhat/kabi/kabi-module/kabi_x86_64/paravirt_ticketlocks_enabled
new file mode 100644
index 0000000..c1ae616
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/paravirt_ticketlocks_enabled
@@ -0,0 +1,2 @@
+#1-
+0xd3719d59 paravirt_ticketlocks_enabled vmlinux EXPORT_SYMBOL
-- 
1.7.1